core/src/main/java/com/scalar/db/storage/jdbc/JdbcAdmin.java

```diff
@@ -67,22 +69,27 @@ public JdbcAdmin(DatabaseConfig databaseConfig) {
     JdbcConfig config = new JdbcConfig(databaseConfig);
     rdbEngine = RdbEngineFactory.create(config);
     dataSource = JdbcUtils.initDataSourceForAdmin(config, rdbEngine);
-    metadataSchema = config.getTableMetadataSchema().orElse(METADATA_SCHEMA);
+    metadataSchema = config.getMetadataSchema().orElse(METADATA_SCHEMA);
```

Comment:
This string is inserted into SQL statements. How about validating this to ensure it doesn't contain any quote or something?

For instance, malicious users could potentially pass `scalar.db.jdbc.metadata.schema=pg_user"; create table foobar (id int); --`.

Reply:
Suppose we sanitize the metadata schema config; doing the same for the keyspace, table, and index creation input parameters makes sense right?
Checking if the input does not contain illegal characters by ourselves seems to be a bad practice as the chance of doing something wrong or too unsafe are high. If we do it, relying on a library seems wiser but I don't know if that exists.
On the other hand, I am not sure if sanitizing the input for the Admin API is the responsibility of ScalarDB. I would think it is more the responsibility of the application using ScalarDB.

Reply:
Oh, I was confused, sorry. `scalar.db.jdbc.metadata.schema` comes from the configuration file and I don't think we need to consider it seriously like input parameters.

As for whether to validate all inputs, it depends. I experienced such a strict validation rule in some enterprise projects. It takes some time and effort, but it reduces uncertainty of systems. I think it's kinda tradeoff.